sdk/resourcemanager/Azure.ResourceManager/src/Custom/Resources/Models/ErrorResponse.Serialization.cs

```diff
@@ -5,12 +5,15 @@
 
 #nullable disable
 
+using System;
 using System.Collections.Generic;
 using System.Text.Json;
+using System.Text.Json.Serialization;
 using Azure.Core;
 
 namespace Azure.ResourceManager.Resources.Models
 {
+    [JsonConverter(typeof(ErrorResponseConverter))]
     public partial class ErrorResponse
```

Comment:
any reason not using IUtf8JsonSerializable?

Reply:
Updated `ErrorResponse.Serialization.cs` and `ErrorAdditionalInfo.Serialization.cs` to use IUtf8JsonSerializable and add the Write methods for them since they are common types and may be used in requests as well. 

For others like `ProviderExtendedLocation` since it is only used in Resources and used for responses only, I will keep the Serialization class unchanged.